common/src/main/java/dev/schmarrn/lighty/event/KeyBind.java

```diff
@@ -44,9 +43,7 @@ public static void handleKeyBind(Minecraft client) {
         KeyBind.oldToggleState = newToggleState;
     }
 
-    public static void init() {
-        ClientTickEvents.END_CLIENT_TICK.register(KeyBind::handleKeyBind);
-    }
+    public static void init() {}
```

Comment:
Putting a comment something like "does nothing besides making sure the keybinds get registered using UtilDefinition"